association_backend/association/api/mutation/types.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import strawberry
+
+
+@strawberry.type
+class FieldError:
+    message: str
+    type: str
+
+
+PydanticError = Optional[list[FieldError]]
```

Comment:
note for myself: this might be part of the shared library code

Reply:
this is now part of pythonit toolkit: [`986605b` (#2084)](https://github.com/pythonitalia/pycon/pull/2084/commits/986605b65ab2be0ca823d5b698c67440686822c1)

Reply:
ok, i removed it because not used anymore